### PR TITLE
Adding filter level documentation to the Objectness filter

### PR DESCRIPTION
**Pull request by blowekamp**

### CODE REVIEW

**Comment by blowekamp** [comment]
The doxygen for this class in SimpleITK is poor:
https://itk.org/SimpleITKDoxygen110/html/classitk_1_1simple_1_1ObjectnessMeasureImageFilter.html
Should address issues raised in discourse:
https://discourse.itk.org/t/what-is-the-standard-deviation-sigma-used-to-compute-the-hessian-in-simpleitks-objectnessmeasure/822

File: include/itkObjectnessMeasureImageFilter.h [hunk 1]
```diff
@@ -23,9 +23,30 @@
...
- * This is a composite filter which combines a computation of the
- * hessian with computation of the objectness.
+ * This filter is a generalization of Frangi's vesselness measurement
+ * for detecting M-dimensional object in N-dimensional space. For
+ * example a vessel is a 1-D object in 3-D space. The filter
+ * can enhance blob-like structures (M=0), vessel-like structures
+ * (M=1), 2D plate-like structures (M=2), hyper-plate-like structures
+ * (M=3) in N-dimensional images, with M<N.
+ *
+ * This filter takes a scalar image as input and produces an real
```
Comment (zivy) [inline include/itkObjectnessMeasureImageFilter.h]: "a" not "an": "produces a real..."

File: include/itkObjectnessMeasureImageFilter.h [hunk 2]
```diff
@@ -23,9 +23,30 @@
...
+ * valued image as output which contains the objectness measure at
+ * each pixel. Internally, it computes a Hessian via discrete central
+ * differences. It is recommended to apply a Gaussian smoothing filter
```
Comment (zivy) [inline include/itkObjectnessMeasureImageFilter.h]: I would suggest changing the wording to:
"Before applying this filter it is expected that a Gaussian smoothing filter at an appropriate scale (sigma) was applied to the input image." 

I don't think this is an optional step (unless you are lucky).

File: include/itkObjectnessMeasureImageFilter.h [hunk 3]
```diff
@@ -23,9 +23,30 @@
...
+ * at a selected scale or sigma to capture the desired object feature.
+ *
```
Comment (zivy) [inline include/itkObjectnessMeasureImageFilter.h]: Please add equations 10,11,12 from Frangi's original paper here. The names alpha for R_a, beta for R_b and gamma for S are fine, but without the equations here users won't know what R_a, R_b, S are and will need to read the original paper (not available to everyone).

Comment (blowekamp) [reply include/itkObjectnessMeasureImageFilter.h]: I'm going to add the equation from the IJ paper.

**Pull request merged**